test/executor.ts

```diff
@@ -1,30 +1,13 @@
 import t from 'tap';
 import { Test } from 'tap';
 
-import { executorTopic as topic } from './helpers/executor';
+import { executorTopic } from './helpers/executor';
 
-t.test('when prompted for a command', async (t: Test) => {
-  t.test('it gets a command', async (t: Test) => {
-    await topic.swear(async ({ executor, host }) => {
-      const prompt = executor.prompt();
-      host.inputStream.write('print "hello world"\n');
+const topic = executorTopic({});
 
-      const command = await prompt;
-      t.matchSnapshot(host.outputStream.output);
-      t.equal(command, 'print "hello world"');
-    });
-  });
-});
-
-t.test('when input is requested', async (t: Test) => {
-  t.test('input is received?', async (t: Test) => {
-    await topic.swear(async ({ executor, host }) => {
-      const question = executor.input('what is your favorite color?');
-      host.inputStream.write('blue\n');
-
-      const input = await question;
-      t.matchSnapshot(host.outputStream.output);
-      t.equal(input, 'blue');
-    });
+t.todo('a meaningful executor test', async (t: Test) => {
```

Comment:
Mocking out input and prompt mean I can't effectively test them. I'm not super torn up about that, but this did show there's an alternative pattern for that which works.

Maybe I can combine the two - don't mock input/prompt, but include a helper that wraps executor and host to make mock I/O easier.